recipes/_solaris2.rb

```diff
@@ -20,6 +20,46 @@
 case node['platform_version'].to_f
 when 5.10
```

Comment:
Yeah, we actually can't use sugar in here because this is the cookbook that installs the toolchain that allows you to install gems that... etc etc. INCEPTION! :) (we could probably make it work, but I'd rather keep this cookbook super small)